cmd/createdkg.go

```diff
@@ -90,6 +98,14 @@ func runCreateDKG(ctx context.Context, conf createDKGConfig) (err error) {
 		}
 	}()
 
+	if len(conf.OperatorENRs) == 0 {
+		return errors.New("no enrs provided with the flag --operator-enrs")
+	}
+
+	if len(conf.OperatorENRs) < conf.Threshold || conf.Threshold < int(math.Ceil(float64(2*len(conf.OperatorENRs)+1)/float64(3))) {
+		return errors.New("insufficient operator ENRs")
```

Comment:
I would say: `threshold too low for number of operators`

Reply:
this if condition also satisfies threshold=3 and no of operators=1